src/grid.cpp

```diff
@@ -547,8 +547,17 @@ diagnostics_t grid::diagnostics(const diagnostics_t &diags) {
 						const real dX[NDIM] = { (x - diags.com[i][XDIM]), (y - diags.com[i][YDIM]), (z - diags.com[i][ZDIM]) };
 						rc.js[i] += dX[0] * U[sy_i][iii] * dV;
 						rc.js[i] -= dX[1] * U[sx_i][iii] * dV;
+						rc.lz2[i] += x * U[sy_i][iii] * dV;
+						rc.lz2[i] -= y * U[sx_i][iii] * dV;
+						rc.lz1[i] += U[lz_i][iii] * dV;
```

Comment:
The difference between lz1 and lz2 is that lz1 also include the spin of each cell. Is that correct?